### PR TITLE
Update sensitive-information-type-learn-about.md

### PR DESCRIPTION
**Pull request by arastogi205**
Adding a note for the current set of solutions that support the improved confidence levels.

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/compliance/sensitive-information-type-learn-about.md [hunk 1]
```diff
@@ -132,6 +132,8 @@ To create custom sensitive information types in the Security & Compliance Center
 
 
 > [!NOTE]
+> Please note that improved confidence levels are available for immediate use within Data Loss Prevention for Microsoft 365 services, Microsoft Information Protection for M365 services, Communication Compliance, Information Governance and Records Management The upgraded confidence levels are available for immediate use
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/sensitive-information-type-learn-about.md]:
```suggestion
> Improved confidence levels are available for immediate use within Data Loss Prevention for Microsoft 365 services, Microsoft Information Protection for Microsoft 365 services, Communication Compliance, Information Governance, and Records Management.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@arastogi205

**Comment by arastogi205** [comment]
i have committed the change. Will this start reflecting on the page?

**Comment by yogkumgit** [comment]
@JohanFreelancer9

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by arastogi205** [comment]
@JohanFreelancer9 - will these changes now reflect on the webpage or does it need an additional approval from someone else?

**Comment by JohanFreelancer9** [comment]
@arastogi205 It still needs to be approved by the document owner before it can go live.

**Comment by yogkumgit** [comment]
@chrfox Please merge this PR. Thanks.

**Pull request merged**